themes/bootstrap3/templates/RecordDriver/EDS/result-list.phtml

```diff
@@ -34,7 +35,28 @@
                   <h2>
                     <a href="<?=$this->escapeHtmlAttr($recordLinker->getUrl($this->driver))?>" class="title getFull _record_link"  data-view="<?=$this->escapeHtmlAttr($this->params->getOptions()->getListViewOption())?>">
                     <?=$item['Data']?> </a>
-                  </h2>
+		  </h2>
+		<?php elseif ('Au' == $item['Group'] && strtolower($edsConfig->AuthorResultListInfo->ResultListPage ?? 'Long') == 'short'): ?>
+                    <?php $authors = $driver->getPrimaryAuthorsWithHighlighting(); ?>
+		    <?php $authorCount = count($authors) ?>
+		    <?php $authorNameLimit = $edsConfig->AuthorResultListInfo->AuthorNameLimit ?? 3; ?>
+                    <?php if (!empty($authors)): ?>
```

Comment:
Since you've already done the work to count the authors, I wonder if it's more efficient to use `$authorCount > 1` here instead of doing an `empty($authors)` check, since `empty` is likely more work.

Reply:
thanks @demiankatz ! Ive included that in my latest follow up